cmd/onboard.go

```diff
@@ -0,0 +1,32 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/operate-first/opfcli/api"
+	"github.com/spf13/cobra"
+)
+
+func NewCmdOnboard(opfapi *api.API) *cobra.Command {
+	cmd := &cobra.Command{
+		Use: "opfcli onboard file-path",
```

Comment:
change : `"opfcli onboard file-path"` to `"onboard file-path"`

Right now the onboard command is `opfcli opfcli file-path`